modules/oadp-about-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: CONCEPT
+[id="oadp-about-backing-and-restoring-from-cluster-to-cluster_{context}"]
+= About backing up data from one cluster and restoring it on another cluster
+
+{oadp-first} is designed to back up and restore application data in the same {product-title} cluster. {mtc-full} ({mtc-short}) is designed to migrate containers, including application data, from one {product-title} cluster to another.
+
+You can use OADP to back up application data from one {product-title} cluster and restore it on another cluster. However, doing so is more complicated than using {mtc-short} or using OADP to back up and restore on the same cluster.
+
+To successfully use OADP to back up data from one cluster and restore it to another cluster, you must take into account the following factors, in addition to the prerequisites and procedures that apply to using OADP to back up and restore data on the same cluster:
+
+* Operators
+* Use of Velero
+* UID and GID ranges
+
+[id="oadp-cluster-to-cluster-operators_{context}"]
+== Operators
+You must exclude Operators from the backup of an application for backup and restore to succeed.
+
+[id="oadp-cluster-to-cluster-velero_{context}"]
+== Use of Velero
+
+Velero, which OADP is built upon, does not natively support migrating persistent volume snapshots across cloud providers. To migrate volume snapshot data between cloud platforms, you must _either_ enable the Velero Restic option `file-system-backup`, which backs up volume contents at the filesystem level, _or_ use the OADP Data Mover for CSI snapshots.
+
+[NOTE]
+====
+The Velero Restic option `file-system-backup` is supported only by OADP 1.2 and later. 
```

Comment:
I think Restic option is supported for OADP 1.1 and OADP 1.0 as well. @kaovilai Please correct me if I'm wrong

Reply:
yes that is correct.. I meant to just say it got renamed in 1.2

Reply:
```suggestion
The Velero Restic option is called `file-system-backup` in OADP 1.2 and later.
```

Reply:
ie. https://velero.io/docs/v1.9/restic/ is now for v1.11 under https://velero.io/docs/v1.11/file-system-backup/

Reply:
Note: @anarnold97 will prepare one or more additional Jiras to be sure the OADP documentation reflects the latest version of Velero.